src/model.rs

```diff
@@ -553,6 +554,24 @@ impl Collections {
             .retain(|address| addresses_used.contains(&address.id));
         self.admin_stations
             .retain(|admin_station| stop_area_ids_used.contains(&admin_station.stop_id));
+        self.occupancies.retain(|occupancy| {
+            if !line_ids_used.contains(&occupancy.line_id) {
+                debug!(
+                    "Occupancy with 'line_id={}' has been removed because stop area is not used",
+                    occupancy.line_id
+                );
+                return false;
+            }
+            if !stop_area_ids_used.contains(&occupancy.from_stop_area) {
+                debug!("Occupancy with 'from_stop_area={}' has been removed because stop area is not used", occupancy.from_stop_area);
+                return false;
+            }
+            if ! stop_area_ids_used.contains(&occupancy.to_stop_area){
```

Comment:
```suggestion
            if !stop_area_ids_used.contains(&occupancy.to_stop_area){
```

Reply:
Thank you. I do not understand why my editor seems to fail reformating on this specific file, only when adding these lines :shrug: :thinking: